src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/NamespaceHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
```

Comment:
Please add `#nullable enable` at the top of the file.

Reply:
done.